src/mpl/src/gavl/mpl_gavl.c

```diff
@@ -304,8 +367,8 @@ int MPL_gavl_tree_search(MPL_gavl_tree_t gavl_tree, const void *addr, uintptr_t
 int MPL_gavl_tree_free(MPL_gavl_tree_t gavl_tree)
 {
```

Comment:
Can we rename tree_free to tree_destroy? I was confused when I saw `tree_free` and `tree_delete` together.